rules/rule-builder.js

```diff
@@ -89,7 +89,8 @@ class RuleBuilder {
             ruleGroup: optionalRuleGroup,
             execute: function (data) {
                 fnToExecute(data);
-            }
+            },
+            tags: this.tags || ['js-rule-builder']
```

Comment:
Why set a default tag here?  I would assume if no tags are supplied, we should not add one ?

Reply:
I thought having the `js-rule-builder` tag always added would be helpful, as you would know where a rule comes from in the UI.